include/alpaka/core/AlignedAlloc.hpp

```diff
@@ -12,51 +12,20 @@
 #include <alpaka/core/BoostPredef.hpp>
 #include <alpaka/core/Common.hpp>
 
-#if BOOST_COMP_MSVC
-#    include <malloc.h>
-#else
-#    include <cstdlib>
+#include <new>
+#if BOOST_COMP_INTEL
+#    include <aligned_new>
```

Comment:
Do we need to remove this in one of your icpc PRs?

Reply:
Yes. Can be removed when icpc is dropped.